content/blog/2023/2023-01-10-jenkins-newsletter.adoc

```diff
@@ -0,0 +1,232 @@
+** Jenkins now has more Java 11 installations than Java 8 installations of Jenkins core! image:/images/post-images/2023/01/10/jenkins-newsletter/image4.png[image,width=403,height=275]
+* Jenkins now fully supports Java 17.
+** Previously Java 17 had been available only in a preview mode, but with the LTS release of 2.361.1, Java 17 functionality is fully available in Jenkins.
+* Migrate Linux installation packages from System V init to systemd.
+** This migration has been requested previously by users since 2017.
+The end result of the migration provides unification of service management implementation and better integration between Jenkins core and service management framework.
+* Staying on top of new backend and frontend dependency updates provides better testing, processing, and performance across Jenkins.
+* Container image updates:
+** We added new platform support such as arm/v7 and aarch64. 
+** The support for ppc64le was removed.
+** The final, definitive version of the containers for JDK8.
+**  The blueocean container image was deprecated.
+** The deprecated install-plugins.sh script was removed from Docker images.
+** There were also "Exit" and "Restart" lifecycle changes in the Docker images.
+* The ANTLR 2 grammars and code were upgraded to ANTLR 4, making it easier for Jenkins to read and parse through various programing languages.
+This also means Jenkins core now compiles with more languages!
+** Thanks to Alex Earl and Basil Crow for all of their hard work on completing this transition!
+** This was also included in Jenkins weekly release 2.376.
+* Platform documentation
+** A short https://www.jenkins.io/doc/administration/requirements/servlet-containers/#sidebar-content[guide] about web containers and servlet container support was created.
+* Platform in progress work:
+** For further development, experiments with RISC-V agents with JDK17/19/20 need to be performed.
+** Further experimenting with Windows 2022 server need to be performed as well.
+
+image:/images/post-images/2023/01/10/jenkins-newsletter/Localization simplification.png[Localization simplification Update]
+
+=== CrowdIn for plugin localization
+Thanks to Alexander Brandes for helping get link:https://crowdin.com/enterprise[CrowdIn] connected with link:https://www.jenkins.io/doc/developer/crowdin/crowdin-integration/#setup-a-crowdin-project[Jenkins].
+This will make the plugin localization process much easier, allowing for any user to translate/localize plugin documentation.
+link:https://crowdin.jenkins.io/[This page] shows the plugins that are currently open and being worked on.
+It also provides some insight as to how many changes have been made and how many people have been contributing to the project.
+![jenkins-crowdin](https://user-images.githubusercontent.com/99040580/211586375-3d4dd4ef-0d58-40a9-94fa-5506b4fbccca.png)
```

Comment:
```suggestion
/images/post-images/2023/01/10/jenkins-newsletter/jenkins-crowdin.png
```
![jenkins-crowdin](https://user-images.githubusercontent.com/99040580/211620813-aefbf223-6bb7-4a16-8c46-da8818027c7b.png)

Reply:
would it be possible to add this image to the folder and update the content here?

Reply:
Yes. 👍